### PR TITLE
Added ability to read downstream and upstream channels

### PR DESCRIPTION
**Pull request by shartte**
Thanks for this project, this saved me a lot of time in figuring out how to get stats from the box :)

Since I've had packet loss (25%+) that were intermittent, I wanted to read error rates from the box and graph them in Grafana, so I extended your lib with reading the upstream/downstream tabs of the ConnectBox interface.

Maybe you'll also find this useful and want to integrate it your project.

### CODE REVIEW

**Comment by pvizeli** [comment]
Convert it to new style please

**Comment by shartte** [comment]
@pvizeli Done

Also, here's a Prometheus reporter script that uses these enhancements to report on the channel error rates / signal to noise ratios to a monitoring tool like Grafana:
https://gist.github.com/shartte/da2b265c1c27c6e00df09556e42a90e6

File: connect_box/__init__.py [hunk 1]
```diff
@@ -74,7 +108,65 @@ def __init__(
...
+                    int(ds.find('freq').text),
+                    int(ds.find('pow').text),
+                    ds.find('mod').text,
+                    ds.find('chid').text,
+                    float(ds.find('RxMER').text),
+                    int(ds.find('PreRs').text),
+                    int(ds.find('PostRs').text),
+                    ds.find('IsQamLocked').text == '1',
+                    ds.find('IsFECLocked').text == '1',
+                    ds.find('IsMpegLocked').text == '1'
```
Comment (pvizeli) [inline connect_box/__init__.py]: use type convert function from attrs

Comment (shartte) [reply connect_box/__init__.py]: Sorry, but I don't want to read up on attrs any further. If this blocks the PR from being merged, please let me know, then I'll close it.

Comment (pvizeli) [reply connect_box/__init__.py]: Okay, but please run tox and fix the issue like the attribute name are not python PEP legal.

Comment (shartte) [reply connect_box/__init__.py]: I fixed the issues tox reported (both style and formatting) and it's not reporting any more issues for me now, although weirdly enough the property names are certainly NOT snake case. Is this check really working correctly?

**Pull request merged**